app/src/main/java/ca/chronofit/chrono/SplashActivity.kt

```diff
@@ -64,6 +64,6 @@ class SplashActivity : BaseActivity() {
     }
 
     companion object {
-        private const val SPLASH_TIMEOUT = 3000L // Timeout delay
+        private const val SPLASH_TIMEOUT = 1500L // Timeout delay
```

Comment:
Should we consider moving this into a resource file? I have a couple other values in mind that can also be rehoused there. Might be easier to make changes as we wont have to hunt down this specific file.
Maybe we can do that as part of the refactoring story.